infra/terraform/test-org/org/locals.tf

```diff
@@ -875,6 +875,16 @@ locals {
       lint_env = {
         ENABLE_BPMETADATA = "1"
       }
+    },
+    {
+      name            = "kms-solutions"
```

Comment:
Is this different from https://github.com/terraform-google-modules/terraform-google-kms?

Reply:
Yeah - It's not a true "tf module", but this repo: https://github.com/GoogleCloudPlatform/kms-solutions

Reply:
Indeed, the one you linked is the terraform KMS module, whereas `kms-solutions` is a new repo where we plan to store samples, artifacts, tutorials, and some terraform blueprints that might not fit the KMS module repo.